tools/distrib/python/grpcio_tools/grpc/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016, Google Inc.
+# Copyright 2015-2016, Google Inc.
```

Comment:
Just the top level files due to namespace package issues that percolate out of us wanting to use auditwheel for our manylinux1 wheels. There might be a magic combination of actions that make user-installing and us testing work simultaneously, but I haven't dug too deep into that. As this is dirty dirty dirty, I prolly' should.